docs/protocol/core/v3.0.rst

```diff
@@ -314,82 +359,111 @@ Core data types
      - Numerical type
      - Size (no. bytes)
      - Byte order
-   * - `bool`
-     - Boolean, with False encoded as `\\x00` and True encoded as `\\x01`
+   * - ``bool``
+     - Boolean, with False encoded as ``\\x00`` and True encoded as ``\\x01``
      - 1
      - None
-   * - `i1`
+   * - ``i1``
      - signed integer
      - 1
      - None
-   * - `<i2`
+   * - ``<i2``
      - signed integer
      - 2
      - little-endian
-   * - `<i4`
+   * - ``<i4``
      - signed integer
      - 4
      - little-endian
-   * - `<i8`
+   * - ``<i8``
      - signed integer
      - 8
      - little-endian
-   * - `>i2`
+   * - ``>i2``
      - signed integer
      - 2
      - big-endian
-   * - `>i4`
+   * - ``>i4``
      - signed integer
      - 4
      - big-endian
-   * - `>i8`
+   * - ``>i8``
      - signed integer
      - 8
      - big-endian
-   * - `u1`
+   * - ``u1``
      - signed integer
      - 1
      - None
-   * - `<u2`
+   * - ``<u2``
      - unsigned integer
      - 2
      - little-endian
-   * - `<u4`
+   * - ``<u4``
      - unsigned integer
      - 4
      - little-endian
-   * - `<u8`
+   * - ``<u8``
      - unsigned integer
      - 8
      - little-endian
-   * - `<f2`
+   * - ``<f2``
      - half precision float: sign bit, 5 bits exponent, 10 bits mantissa
      - 2
      - little-endian
-   * - `<f4`
+   * - ``<f4``
      - single precision float: sign bit, 8 bits exponent, 23 bits mantissa
      - 4
      - little-endian
-   * - `<f8`
+   * - ``<f8``
      - double precision float: sign bit, 11 bits exponent, 52 bits mantissa
      - 8
      - little-endian
-   * - `>f2`
+   * - ``>f2``
      - half precision float: sign bit, 5 bits exponent, 10 bits mantissa
      - 2
      - big-endian
-   * - `>f4`
+   * - ``>f4``
      - single precision float: sign bit, 8 bits exponent, 23 bits mantissa
      - 4
      - big-endian
-   * - `>f8`
+   * - ``>f8``
      - double precision float: sign bit, 11 bits exponent, 52 bits mantissa
      - 8
      - big-endian
+   * - ``r*`` (Optional)
+     - raw bits,  use for extension type fallbacks
+     - variable, given by ``*``, is limited to be a multiple of 8.
+     - N/A
```

Comment:
I think this is too confusing to describe it as bits. The size column is in bytes, and we are saying it is limited to be a multiple of 8 bits anyway. I think we should just say "raw bytes".

Reply:
Another reason is that all the other identifiers have a value in number of bytes, e.g. `i1` which is a 1-byte integer.

Reply:
Sure, but then we'll never be able to go lower or non multiple of 8.